sdk/error.go

```diff
@@ -937,7 +970,7 @@ func (e *MultiError) Append(err error) {
 			e.Append((*mError)[i])
 		}
 	} else {
-		*e = append(*e, err)
+		*e = append(*e, WithStack(err))
```

Comment:
include the multiError Append in the stack of each function may be annoying

Reply:
I added a filter on func name in the print of the log call stack to ignore it.